packages/react-composites/src/composites/CallWithChatComposite/adapter/AzureCommunicationCallWithChatAdapter.test.ts

```diff
@@ -38,15 +38,13 @@ describe('Adapter is created as expected', () => {
     const locator = { meetingId: '123', passcode: 'qwe' };
     const statefulChatClient = createStatefulChatClientMock(new StubChatThreadClient());
     const threadClient = statefulChatClient.getChatThreadClient('threadId');
-    /* @conditional-compile-remove(teams-inline-images-and-file-sharing) */
```

Comment:
why does it look like we are removing these lines in this PR?

Reply:
+1 to this question

Reply:
I've spoke Ruslan here, and the reason why this cc code is being removed it because:
1. Ruslan created branch for meeting id for composite
2. Leah removed teams-inline-images as seen in this [commit/pr #4305 ](https://github.com/Azure/communication-ui-library/commit/a04d436a8ccb7c6f5ef9194f25baaf17e44fede0)
3. Ruslan missed Leahs pr so Ruslan's pr was behind main changes. 
4. Ruslan had to use teams-inline-images to compile his tests on his branch that was behind main
5. Ruslan merged his changes to main, no errors/warnings were seen from the cc codes he added before updating his branch
6. Now, in this new branch, he is removing the cc code he added that was merged by mistake after Leahs removal